snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -169,9 +175,20 @@ def initialize_replica(self,
                 raise error
         self._init_image(source_adapter_name)
 
-    def _init_image(self, 
+    def _init_image(self,
                     source_adapter_name: str) -> None:
         shdocker = SnowShuDocker()
+
+        image_architecture = shdocker.get_docker_image_attributes(self.DOCKER_IMAGE) \
+            .get('Architecture', LOCAL_ARCHITECTURE)
```

Comment:
Is there are specific case when image attribute "Architecture" has no value, so we snould to specify `LOCAL_ARCHITECTURE` as default value?